### DISCUSSION TITLE
bluetooth midi devices on minidexed

### DISCUSSION THREAD

**Discussion by mongio** [probonopd/MiniDexed#402] [Ideas]
Hello, 
my minidexed runs on a Rpi 4b, which has bluetooth, my digital piano has a midi bluetooth dongle (Yamaha BT01) which I use for connecting it to my ipad, and it works very well. I wonder if bluetooth midi could be implemented in minidexed, it would be very useful.
Thanks
Maurizio

**Comment by probonopd** [comment]
Hi @mongio

https://github.com/rsta2/circle/issues/203

> There was some rudimentary BT support in Circle earlier, but it was removed for legal reasons. 

Please see that thread for details.
